doc/classes/EditorSettings.xml

```diff
@@ -880,6 +880,9 @@
 		<member name="project_manager/default_renderer" type="String" setter="" getter="">
 			The renderer type that will be checked off by default when creating a new project. Accepted strings are "forward_plus", "mobile" or "gl_compatibility".
 		</member>
+		<member name="project_manager/directory_naming_convention" type="int" setter="" getter="">
+			Directory naming convention for the project manager. Options are "No convention" (project name is directory name), "kebab-case" (default), "snake_case", "camelCase", "PascalCase", or "Title Case".
```

Comment:
Right now, the default is kebab-case and it replaces underscores: **`t e_s-t` → `t-e-s-t`**
It would probably be better to keep underscores instead: **`t e_s-t` → `t-e_s-t`**

Could be implemented like this: `dir_name = project_name.to_lower().replace(" ", "-")`

Reply:
The default (kebab-case) is now implemented as `to_lower().replace(" ", "-")`. So underscores are preserved: **`t e_s-t` → `t-e_s-t`**